FailPrompt.py

```diff
@@ -2,46 +2,51 @@
 
 # Commands
 def instructions():
-	print("Type a command, then press enter.")
+    print("Type a command, then press enter.")
 
 def ping():
-	print("Pong!")
+    print("Pong!")
 
 def matrix():
     for i in range(0, 1000000):
         print(random.randrange(0, 9), end = '')
 
-def quit():
-	keepGoing = False
-
+# Variables
 commands = { "instructions": instructions, 
-			 "ping": ping,
-             "matrix": matrix,
-			 "quit": quit }
```

Comment:
The quit command is more of a framework command than an actual command. As such, it's better for it to not be here.